lib/EventHandler.js

```diff
@@ -23,6 +24,13 @@ module.exports = class EventHandler {
     this._kubeResourceMeta = params.kubeResourceMeta;
     this._kc = params.kubeClass;
     this._factory = params.factory;
+    if (params.livenessInterval) {
```

Comment:
non necessary existence check

Reply:
Not necessary but avoids constructing a string to be garbage collected later when its not set